packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_market.js

```diff
@@ -49,10 +50,35 @@ Blockly.Blocks.trade_definition_market = {
 
         this.enforceLimitations();
 
+        const forexHideList = [
+            'gb',
+            'ie',
+            'pt',
+            'se',
+            'pl',
+            'cy',
+            'nl',
+            'si',
+            'lv',
+            'hu',
+            'cz',
+            'ro',
+            'dk',
+            'fi',
+            'lt',
+            'hr',
+            'ee',
+            'sk',
+            'bg',
+            'at',
+        ];
```

Comment:
We have a config file to store this kind of const it's better if you move it there
`deriv-app/packages/bot-skeleton/src/constants/config.js`

Reply:
@mehrzad-fs done